backends/advanced/src/advanced_omi_backend/cron_scheduler.py

```
@@ -0,0 +1,277 @@
+            except asyncio.CancelledError:
+                pass
+        if self._redis:
+            await self._redis.close()
+        logger.info("Cron scheduler stopped")
+
+    # -- public API ----------------------------------------------------------
+
+    async def run_job_now(self, job_id: str) -> dict:
+        """Manually trigger a job regardless of schedule."""
+        if job_id not in self.jobs:
+            raise ValueError(f"Unknown cron job: {job_id}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)